src/howtos/custom-routes.md

```diff
@@ -0,0 +1,42 @@
+# Custom Routes
+
+Epinio has the concept of a "system domain". This domain is set when installing
+Epinio (with the `domain` helm value). It must be a [wildcard domain](https://en.wikipedia.org/wiki/Wildcard_DNS_record) and is used in 3 different cases:
+
+- To create a URL for the Epinio API server
```

Comment:
```suggestion
- To create an URL for the Epinio API server
```

Reply:
I think a/an is chosen based on the "sound" of the next word not whether is a vowel or a consonant: https://whatis.techtarget.com/feature/Which-is-correct-a-URL-or-an-URL

Reply:
Ok. I had it sounded out as `an url`, not as `a you-are-ell`, i.e. I did no speak spelling.